bos_mint/config-defaults.yaml

```diff
@@ -3,15 +3,16 @@ debug: False
 project_name: MINT
 project_sub_name: The BOS Manual Intervention Module
 secret_key: # enter any random string 
+advanced_features: True
 
 sql_database: "sqlite:///{cwd}/bookied-local.db"
 
 connection:
-    use: # enter your desired chain
+    use: elizabeth # enter your desired chain
```

Comment:
this is default config and should nto work without production config, please leave values as before